io/Cargo.toml

```
@@ -4,6 +4,7 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-gstd = { git = "https://github.com/gear-tech/gear.git" }
-scale-info = { version = "2.2.0", default-features = false, features = ["derive"] }
-codec = { package = "parity-scale-codec", version = "3.1.2", default-features = false, features = ["derive"] }
+gstd = { git = "https://github.com/gear-tech/gear.git", features = ["debug"] }
+codec = { package = "parity-scale-codec", version = "3", default-features = false, features = ["derive"] }
```

No need for features.
```suggestion
codec = { package = "parity-scale-codec", version = "3", default-features = false }
```